src/m365/spo/commands/apppage/apppage-set.ts

```diff
@@ -69,7 +69,7 @@ class SpoAppPageSetCommand extends SpoCommand {
       },
       responseType: 'json',
       data: {
-        serverRelativeUrl: `${urlUtil.getServerRelativePath(args.options.webUrl, '')}/SitePages/${args.options.name}`,
+        serverRelativeUrl: `${urlUtil.getServerRelativePath(args.options.webUrl, '').replace(/\/$/, '')}/SitePages/${args.options.name}`,
```

Comment:
Let's not use regex for replacement though. That's an overkill here that's also less readable.

Reply:
hmm, on second thought. I now see what you're doing here: only replacing the last character if its a slash. Let's keep it as is 👍

Reply:
Thinking some more on it @saurabh7019. I see what you're doing here with `$`, only replacing the last character if it's a /. It works like a charm. The big but is readability. And come to think of it: the `getServerRelativePath` function also checks for trailing slashes and double slashes itself. 

So we can change the code to a much simpler version:

`${urlUtil.getServerRelativePath(args.options.webUrl, 'SitePages')}/${args.options.name}`

Moving the FolderRelativeUrl `SitePages` as the second argument of the `getServerRelativePath` function.

Reply:
Neat! I will make this change.